contracts/SpeedMarkets/SpeedMarketsAMM.sol

```diff
@@ -71,39 +72,26 @@ contract SpeedMarketsAMM is Initializable, ProxyOwned, ProxyPausable, ProxyReent
     mapping(address => AddressSetLib.AddressSet) internal _activeMarketsPerUser;
     mapping(address => AddressSetLib.AddressSet) internal _maturedMarketsPerUser;
 
-    struct MarketData {
```

Comment:
unsure if this is upgrade safe, try it on goerli first

Reply:
Successfully upgraded on goerli